install/check-memcached-backend.sh

```diff
@@ -0,0 +1,17 @@
+echo "${_group}Checking memcached backend ..."
+
+if grep -q "\.PyMemcacheCache" "$SENTRY_CONFIG_PY"; then
+  echo "found PyMemcacheCache in $SENTRY_CONFIG_PY, looks good"
+else
+  if grep -q "\.MemcachedCache" "$SENTRY_CONFIG_PY"; then
+    echo "found MemcachedCache in $SENTRY_CONFIG_PY, switching to PyMemcacheCache"
+    sed -i '' -e 's/\.MemcachedCache/.PyMemcacheCache/' "$SENTRY_CONFIG_PY"
+    sed -i '' -e 's/^\( *\)"server_max_value_length.*$/\1"ignore_exc": True,/' "$SENTRY_CONFIG_PY"
```

Comment:
What if they have more configs on this line? This will erase those. How about:

```suggestion
    sed -i '' -e 's/^\( *\)"server_max_value_length.+?\),/\1"ignore_exc": True,/' "$SENTRY_CONFIG_PY"
```

But I think we could avoid all of this search and replace if we took the `CACHES_BAK` route above, and just wrote the new config wholesale.